### ISSUE TITLE
Some lines vanish from postfix.cf override

### ISSUE THREAD

**Issue by Al2Klimov** [Mailu/Mailu#1827]
Thank you for opening an issue with Mailu. Please understand that issues are meant for bugs and enhancement-requests.
For **user-support questions**, reach out to us  on [matrix](https://matrix.to/#/#mailu:tedomum.net).

To be able to help you best, we need some more information.

## Before you open your issue
- [x] Check if no issue or pull-request for this already exists.
- [x] Check [documentation](https://mailu.io/master/) and [FAQ](https://mailu.io/master/faq.html). (Tip, use the search function on the documentation page)
- [x] You understand `Mailu` is made by volunteers in their **free time** — be conscise, civil and accept that delays can occur.
- [x] The title of the issue should be short and simple. It should contain specific terms related to the actual issue. Be specific while writing the title.

### Environment
 - [x] docker-compose
 - [ ] kubernetes
 - [ ] docker swarm

### Versions
To find your version, get the image name of a mailu container and read  the version from the tag (example for version 1.7).
```
$> docker ps -a | grep mailu
140b09d4b09c    mailu/roundcube:1.7    "docker-php-entrypoi…"    2 weeks ago    Up 2 days (healthy)    80/tcp
$> grep MAILU_VERSION docker-compose.yml mailu.env
```

1.7

## Description
Further explain the bug in a few words. It should be clear what the unexpected behaviour is.  Share it in an easy-to-understand language.

See below.

## Replication Steps
Steps for replicating your issue

1. Create an `/overrides/postfix.cf` like below (or [here](http://www.postfix.org/RESTRICTION_CLASS_README.html))
2. restart containers
3. cat `/etc/postfix/main.cf` – only the first overridden line appears

```
smtpd_recipient_restrictions =
  check_recipient_address pcre:/overrides/check_recipient_address
```

## Expected behaviour
Explain what results you expected - be as specific as possible. Just saying "it doesn’t work as expected" is not useful. It's also helpful to describe what you actually experienced.

Both lines are there.

## Logs
Often it is very useful to include log fragments of the involved component. You can get the logs via `docker logs <container name> --tail 1000`. For example for the admin container:
`docker logs mailu_admin_1 --tail 1000`
or using docker-compose `docker-compose -f /mailu/docker-compose.yml logs --tail 1000 admin`
If you can find the relevant section, please share only the parts that seem relevant. If you have any logs, please enclose them in code tags, like so:
````markdown
```
Your logs here!
```
````

**Comment by Al2Klimov** [comment]
Ok, the directive should be _check_recipient_access_, but that doesn’t fix the problem.

**Comment by Diman0** [comment]
For both ``postfix.cf`` and ``postfix.master``, you need to put one configuration per line, as they are fed line-by-line
     to postfix. Did you do this?
You might also be effected by [this issue](https://github.com/Mailu/Mailu/pull/1654) which is addressed in mailu 1.8.

**Comment by Al2Klimov** [comment]
Ok, line-by-line works.
